src/main/webapp/js/userMap.js

```diff
@@ -14,20 +14,25 @@ function handleData(err, countryCoordinates, userData) {
         handleError();
         return;
     }
-    var userCountries = Object.keys(userData);
     var countriesArr = [];
     var total = 0;
-    userCountries.forEach(function(countryName) {
-        var countryTotal = userData[countryName].length;
-
-        countriesArr.push([countryName, countryTotal]);
-        total += countryTotal;
-    });
-
+    var countryTotal = 0;
+    var date = userData.lastUpdated;
+    for (key in userData.institutes) {
```

Comment:
I don't know what is wrong with `userData.institutes[countryName].length;` but earlier i tried this only. But i am not getting the expected answer ( may be because the json structure was disturbed). So after trying a lot i have moved to this.